nullaway/src/main/java/com/uber/nullaway/NullAway.java

```diff
@@ -815,15 +815,21 @@ private boolean fieldAlwaysInitializedBeforeRead(
         // Hack: Handling try{...}finally{...} statement, see getSafeInitMethods
         if (curStmt.getKind().equals(Tree.Kind.TRY)) {
           TryTree tryTree = (TryTree) curStmt;
+          // ToDo: Should we check initialization inside tryTree.getResources ? What is the scope of
+          // that initialization?
```

Comment:
This is actually important if we are supporting code with `try-with-resources` in general, but doing this PR first will at least fix up the crash, which is higher priority.